Idno/Core/Purifier.php

```diff
@@ -14,8 +14,22 @@ function init()
                 $upload_dir = \Idno\Core\Idno::site()->config()->getTempDir();
             }
 
+            $allowedIframes = [
+                'www.youtube.com/embed/',
+                'player.vimeo.com/video/',
+                'embed.radiopublic.com/e',
+                'w.soundcloud.com/player/',
+                'maps.google.com/'
+            ];
+
+            if (!empty(Idno::site()->config()->allowedIframes) && is_array(Idno::site()->config()->allowedIframes)) {
+                $allowedIframes = array_merge($allowedIframes, Idno::site()->config()->allowedIframes);
```

Comment:
I'd be tempted to wrap this in an event trigger, that way plugins can easily extend this. Otherwise lg

Reply:
:+1: I'll follow up with that